osvr/RenderKit/RenderManager.h

```diff
@@ -332,14 +332,28 @@ namespace renderkit {
         /// room transform, but will have the room appended.
         class RenderParams {
           public:
-            RenderParams(){};
+            /// @brief Default constructor reads things from a display configuration.
+            /// @param [in] dc OSVRDisplayConfiguration to read values from.  If none
+            ///                is specified, it defaults to the default configuration.
+            RenderParams(OSVRDisplayConfiguration const &dc = OSVRDisplayConfiguration()) {
+              nearClipDistanceMeters = dc.getNearClipMeters();
+              farClipDistanceMeters = dc.getFarClipMeters();
+              IPDMeters = dc.getIPDMeters();
```

Comment:
I'm assuming this call is to effectively just a constant return value at this time, since I don't see changes to that class in this PR, is this right?